packages/editor/src/components/resizable-editor/resize-handle.js

```diff
@@ -15,6 +15,11 @@ export default function ResizeHandle( { direction, resizeWidthBy } ) {
 	function handleKeyDown( event ) {
 		const { keyCode } = event;
 
+		if ( keyCode !== LEFT && keyCode !== RIGHT ) {
+			return;
+		}
+		event.preventDefault();
```

Comment:
Should we add a comment referencing this issue, or do you think it's a common practice? I think maybe it's fine to leave it to git blaming to do its work 😅.

Reply:
I think if we want to add our own event to an event, it is common to add `event.preventDefault()`, and there are a lot of uncommented `event.preventDefault()` in Gutenberg 😅 Therefore, I personally think that a comment is unnecessary.